### PR TITLE
feat add history page and a glance of recent 5 transactions

### PR DESCRIPTION
**Pull request by ruveyda-kisla**
### Description
Issue #16 
- Implement History page and a glance of recent 5 transactions.
- Implement History details modal and Filter.
- Implement `CCheckBox`.
- Implement `CStatusCard`.
- Fix z-index of `Aside` so Modal would drop a shadow on it.
- Add Dashboard content from `src/app/dashboard` to `src/containers/dashboard`.
- Change `productList` to `mockList` and add all the mock data we temperately need to it.
- Change some style differences in `Products` page.
- Change Aside Icons.
- Change `CStausCard` to `CChartSummery`.
- Change `CProductItemCard` Type to `IProductItemCard`.

### Checklist

- [x] Tested changes locally.
- [x] Ensured code follows project standards.
- [x] Updated documentation.
- [ ] Added necessary tests.

### Additional Notes

This branch will have conflicts with the other [PR](#15) but I will fix them after that one is merged.

### CODE REVIEW

File: src/components/CChartSummery/index.tsx [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+          'bg-transparent': transparent,
+        },
+        { 'bg-[#F2F4F7]': !transparent },
+      )}
+    >
+      {text ? (
+        <div className="text-[#101828]">{text}</div>
+      ) : (
+        percentage && (
+          <div
```
Comment (ceydae) [inline src/components/CChartSummery/index.tsx:44]: unacceptable code flow here. Fix it
The problem:

File: src/containers/HistoryItemCard/Modal/index.tsx [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+  token,
+  image,
+  method,
+  status,
+  isOpen,
+  amount,
+  sender,
+  onClose,
+  progress,
+  cancellableAfter,
```
Comment (ceydae) [inline src/containers/HistoryItemCard/Modal/index.tsx:32]: too many arguments. This is also bad code. fix it

File: src/containers/HistoryItemCard/index.tsx [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+  date,
+  title,
+  token,
+  image,
+  method,
+  status,
+  amount,
+  sender,
+  progress,
+  cancellableAfter,
```
Comment (ceydae) [inline src/containers/HistoryItemCard/index.tsx:24]: too many arguments. bad code

**Review by ceydae — CHANGES_REQUESTED** [review]

**Review by dousdogukan — APPROVED** [review]

File: src/components/CChartSummery/index.tsx (hunk 1 above)
Comment (ruveyda-kisla) [reply src/components/CChartSummery/index.tsx:44]: not my code

**Pull request merged**